### DISCUSSION TITLE
Flag when taxon does not equal prefered taxon

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5867] [Feature Requests]
>When entering determinations it would be great if there could a flag/warning to let the user know when the teaxon that they have entered does not match the prefered taxon. This would make it easier to see that the name they are using is a synonym.
>
>Our team has suggested that the preferred taxon field changing colour would be a good way to flag this.
>
> If this could be a user preference that would be great.

Some other [discourse takes place on the forum](https://discourse.specifysoftware.org/t/flag-when-taxon-does-not-equal-prefered-taxon/1068/5) that may be useful when deciding on implementation.

> We record multiple determinations for our specimens, including label name, determinations, fide, typification. We use the ‘Is Current’ flag to indicate which of the several determinations a specimen could have to indicate which one the specimen should be filed under. In general we want to try and avoid filing a specimen under a synonym.
> 
> We have the preferred taxon underneath the taxon field, so it can be checked, however when processing large numbers of specimens it can be easy to miss that there is a difference in the two fields.
> 
> I agree that for certain workflows it may be that this could be confusing, whichi is why a preference that could be enabled/disabled may be the way forward, so that it can be set by individual users/institutions.

**Requested By:** [@rdrinkwater on the Speciforum](https://discourse.specifysoftware.org/t/flag-when-taxon-does-not-equal-prefered-taxon/1068/5)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/flag-when-taxon-does-not-equal-prefered-taxon/1068/7
